File: testing/unit-test-patterns/javascript/src/common/models/Clone.ts
import { IClone } from '../interfaces/Clone.interface'
import { Gender } from './gender'

export class Clone implements IClone {
  constructor(
    firstName: string,
    lastName: string,
    gender: Gender,
    hairColor: string,
    numberOfEyes: number
  ) {
    this.firstName = firstName
    this.lastName = lastName
    this.gender = gender
    this.hairColor = hairColor
    this.numberOfEyes = numberOfEyes
  }

  firstName: string
  lastName: string
  gender: Gender
  hairColor: string
  numberOfEyes: number
}

File: testing/unit-test-patterns/c#/UnitTestPatterns/Common/Models/Clone.cs
using UnitTestPatterns.Common.Interfaces;

namespace UnitTestPatterns.Common.Models
{
    public class Clone : IClone
    {
        public Clone(string firstName, string lastName, Gender gender, string hairColor, int numberOfEyes)
        {
            FirstName = firstName;
            LastName = lastName;
            Gender = gender;
            HairColor = hairColor;
            NumberOfEyes = numberOfEyes;
        }

        public string FirstName { get; }
        public string LastName { get; }
        public Gender Gender { get; }
        public string HairColor { get; }
        public int NumberOfEyes { get; }
    }
}